src/serviceDesks/incontact/inContactServiceDesk.ts

```diff
@@ -103,20 +103,6 @@ class InContactServiceDesk implements ServiceDesk {
       navigator.sendBeacon(`${this.SERVER_BASE_URL}/incontact/end`, JSON.stringify(this.session)); // https://golb.hplar.ch/2018/09/beacon-api.html
     });
 
-    try {
-      const request = await fetch(`${this.SERVER_BASE_URL}/incontact/queue`, {
-        method: 'POST',
-        headers: { Accept: 'application/json', 'Content-Type': 'application/json' },
-        body: JSON.stringify({ ...this.session }),
```

Comment:
You can probably just use `this.session` instead of `{ ...this.session }`. The spread isn't getting you anything.

Reply:
I'm not sure why you commented on this section because this is deleted code? But will apply these comments for anywhere else I see similar issues.